lib/activerecord-import/adapters/mysql_adapter.rb

```diff
@@ -32,21 +34,26 @@ def insert_many( sql, values, *args ) # :nodoc:
 
     # if we can insert it all as one statement
     if NO_MAX_PACKET == max || total_bytes < max
-      number_of_inserts += 1
+      number_of_inserts = values.size
```

Comment:
This should be reverted to `number_of_inserts += 1` because it is not counting the number of rows inserted, but the actual number of insert operations executed.